keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -343,35 +343,86 @@ are properly reloading tokens by:
 1.  Add annotation to audit events for legacy and stale tokens including
     necessary information to locate problematic client.
 
```

Comment:
above, note the `--service-account-extend-token-expiration` mitigation defaults to true, and that cluster administrators can set it to `--service-account-extend-token-expiration=false` to turn off the mitigation if desired

Reply:
done